scopes/defender/formatter/formatter.docs.mdx

```diff
@@ -0,0 +1,8 @@
+---
```

Comment:
A unified interface for formatting components

Reply:
done